installing/installing-mirroring-installation-images.adoc

```diff
@@ -65,6 +65,37 @@ In production environments, add the required images to a registry in your restri
  modules/installation-performing-disconnected-mirror-without-registry.adoc[leveloffset=+2]
 ////
 
+[id="mirror-registry"]
+== Mirror registry for {product-title}
+
+This section introduces how to install mirror registry for {product-title}, a small and streamlined container registry to use as a target for mirroring the required container images of {product-title} for disconnected installations.
+
+.Prerequisites
+
+* An {product-title} subscription.
+* {op-system-base-full} 8 or Fedora CoreOS with Podman 3.3 installed.
+* About 12 GB for {product-title} 4.10 Release images, or about 358 GB for {product-title} 4.10 Release images and {product-title} 4.10 Red Hat Operator images.
+* Fully qualified domain name for the Red Hat Quay service, which must resolve through a DNS server or your `/etc/hosts` file.
```

Comment:
`/etc/hosts` on the system running Quay, or an 'external' system? 
I think its simpler to just require an FQDN in / from a proper dns system, I recommend us striking the `/etc/hosts` recommendation.